### ISSUE TITLE
Feature Request: Munki module reports server in listings

### ISSUE THREAD

**Issue by bryanzak** [munkireport/munkireport-php#1417]
In the Munki module that reports munki information in Listings such as 

Name, Serial Number, Username, IP address, OS Version, Munki Version, Last Seen, Run Type, Errors, Warnings, Manifest Name

It would be very useful to see the munki server (Software Repo URL) as well as we are a large organization with many munki servers and having a single report that told us which server all of our computers were connected to would be very useful

**Comment by flammable** [comment]
If you have the `munkiinfo` module enabled, you can sort by `SoftwareRepoURL`.

**Comment by bryanzak** [comment]
We do, but I don't see that field?

<img width="2115" alt="Munki Info Listing" src="https://user-images.githubusercontent.com/5674647/125322147-0dbf9b00-e303-11eb-8fff-bad217543d78.png">

**Comment by bryanzak** [comment]
And this is the Munki module (confusingly named MunkiReport?) where having the softwarerepo listed would be useful
<img width="2101" alt="Munki Listing" src="https://user-images.githubusercontent.com/5674647/125322329-4495b100-e303-11eb-902f-2f5b89cee492.png">

**Comment by gmarnin** [comment]
It's in `Munki Info` listing https://github.com/munkireport/munkiinfo

**Comment by bryanzak** [comment]
Ah yes, we have that enabled. So maybe my ignorance is that I don't understand how to customize the fields that the Munki Info Listing returns? As you can see above we are only seeing Name, Serial Number, username, Key, Value, Last Seen. Is there a way to customize that Listings columns that I am missing?

**Comment by gmarnin** [comment]
It looks like it should be the `CatalogURL` key but it's not populated in my listing either. I do see the address populated on the Client Details page on the Munki section as `Software Repo URL` so the address is in the DB.

**Comment by bryanzak** [comment]
yep - I see it just fine on Client Details, but not Listings where it would be particularly useful for us

**Comment by bryanzak** [comment]
it really feels like the munkiinfo listings is .. incomplete? I would have expected all (or at least most) of the filelds to be reported but they aren't

**Comment by flammable** [comment]
I'm looking in Listings > Munki Info. I can put `SoftwareRepoURL` in the search box, and it returns the Munki repo URL set via our config profile.

If you're seeing some of those clients are returning empty data, it's possible the module wasn't enabled when you built your MunkiReport client pkg. Try building a new one and deploying it ([here's a good method of doing that with AutoPkg](https://github.com/munkireport/munkireport-php/wiki/AutoPkg#version-override)).

**Comment by gmarnin** [comment]
I see `SoftwareRepoURL` is populated. I was looking at the wrong key in the listings view

**Comment by bryanzak** [comment]
Oh! I had no idea the Key field worked that way!  Wowza - are there other Listings where something like that works?

I thought the search box could only search what was shown in the table (see my earlier screenshot).

**Comment by bryanzak** [comment]
OK nvm I see what is going on, ok now it makes sense. I'd still prefer SoftwareRepoURL be in the Munki module as a column just like manifest, but this is better than nothing and allows me to extract the info I need. Thanks

**Comment by gmarnin** [comment]
You could gen a report using the API to read this key
